docs/iris/src/userguide/interpolation_and_regridding.rst

```diff
@@ -73,19 +90,18 @@ We can interpolate specific values from the coordinates of the cube:
 As we can see, the resulting cube is scalar and has longitude and latitude coordinates with
 the values defined in our sample points.
 
-It isn't necessary to specify sample points for each dimension - any dimensions which aren't
+It isn't necessary to specify sample points for each dimension. Any dimensions that aren't
```

Comment:
Perhaps ... `... for each dimension, only those that you want to interpolate over.`